src/utils/staking.js

```diff
@@ -0,0 +1,278 @@
+import * as nearApiJs from 'near-api-js'
+import sha256 from 'js-sha256';
+import BN from 'bn.js'
+
+const {
+    transactions: {
+        functionCall
+    },
+    utils: {
+        format: {
+            parseNearAmount
+        }
+    }
+} =  nearApiJs
+
+// 125 - 175 Tgas for lockup methods
+const GAS_STAKE = '175000000000000'
```

Comment:
lockup contract docs specify gas for every operation as multiples if certain base cost (25Tgas I think). It's best if we also do it and have base cost overridable in env var